src/main/containers/DataCanvas.jsx

```diff
@@ -13,7 +13,7 @@ const DataCanvas = (props) => {
     return (
       <EmptyState>
         <h2>Let’s get some data</h2>
-        <p>To get started, send a request to any of your microservices—or open a data mockup</p>
+        <p>To get started, send a request to any of your microservices <br/>or post http://localhost:3001/posturl</p>
```

Comment:
During our UI cleanup, I’d love to give this empty state some more thought as a team—just putting as a reminder here.